modules/beamdyn/src/Driver_Beam.f90

```diff
@@ -98,8 +98,7 @@ PROGRAM BeamDyn_Driver_Program
       
       ! initialize the BD_InitInput values not in the driver input file
    BD_InitInput%RootName = TRIM(BD_Initinput%InputFile)
-   BD_InitInput%RootDisp = matmul(transpose(BD_InitInput%RootOri),BD_InitInput%GlbPos) - BD_InitInput%GlbPos
-   BD_InitInput%RootVel(1:3) = matmul(BD_InitInput%RootOri, Cross_Product( BD_InitInput%RootVel(4:6), BD_InitInput%GlbPos ))  ! set translational velocities based on rotation and GlbPos.
```

Comment:
By removing the rotational velocity input, how are you doing the initialization of a rotating blade?  If there is a hub radius, these terms will be non-zero from the OpenFAST glue code.

Reply:
Rotational velocity is initialized in BD_ReadDvrFile:

`   InitInputData%RootVel(1:3) = cross_product(InitInputData%RootVel(4:6),InitInputData%GlbPos(:))`

Reply:
Ok.  This makes sense to me now.